Generate.py

```diff
@@ -256,18 +333,217 @@ def get_choice_legacy(option, root, value=None) -> Any:
     raise RuntimeError(f"All options specified in \"{option}\" are weighted as zero.")
 
 
-def get_choice(option, root, value=None) -> Any:
-    if option not in root:
-        return value
-    if type(root[option]) is list:
-        return random.choices(root[option])[0]
-    if type(root[option]) is not dict:
-        return root[option]
-    if not root[option]:
-        return value
-    if any(root[option].values()):
-        return random.choices(list(root[option].keys()), weights=list(map(int, root[option].values())))[0]
-    raise RuntimeError(f"All options specified in \"{option}\" are weighted as zero.")
+BinOp = Literal["+", "-", "*", "/"]
+
+
+_bops: Mapping[BinOp, Callable[[Union[float, int], Union[float,int]], Union[float, int]]] = {
+    "+": lambda a, b: a + b,
+    "-": lambda a, b: a - b,
+    "*": lambda a, b: a * b,
+    "/": lambda a, b: a / b if isinstance(a, float) or isinstance(b, float) else a // b,
```

Comment:
I don't think it's a good idea to force users to understand integer division.

Users will understand when an integer is needed for the final result. They will expect `"5 / 2 * 100"` to give the integer 250, and will be confused when it gives 200.

I think it's better to do all the calculations as floats, and then, if an integer is needed for the end result, use rounding or truncate.

Reply:
(I think most of the `Option`s already call `int()` on whatever we pass to them.)